.github/workflows/pages.yml

```
@@ -0,0 +1,18 @@
+on:
+  # TODO: Remove me once the PR that adds docs is merged
+  pull_request: {}
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)